### PR TITLE
Do not include yaml related targets if fletch does not have yaml

### PR DESCRIPTION
**Pull request by aaron-bray**
We still need to work out where this lives

### CODE REVIEW

**Comment by dstoup** [comment]
Looks reasonable and will be a useful branch to merge since it fixes one of the current dashboard issues.

File: vital/kpf/CMakeLists.txt [hunk 1]
```diff
@@ -2,6 +2,13 @@
 # KPF parsing
 #
 
+if(fletch_ENABLED_YAMLCPP)
+  find_package( yaml-cpp REQUIRED )
+else()
+  message(STATUS "To enable kpf support, please build or located yaml libraries in fletch")
```
Comment (linus-sherrill) [inline vital/kpf/CMakeLists.txt]: Should this be "locate"?

**Comment by aaron-bray** [comment]
yes

**Comment by dstoup** [comment]
Jenkins test this please

**Pull request merged**